ecs_service/ecs_task_definition.tf

```
@@ -0,0 +1,58 @@
+data "aws_secretsmanager_secret" "app" {
+  name = "${var.project}/app/${var.environment}"
```

```suggestion
  name = var.secrets_alias === null ? "${var.project}/app/${var.environment}" : var.secrets_alias
```

could we do sth like that to allow passing a alias that differs from the standard?